include/fiction/io/write_sqd_sim_result.hpp

```diff
@@ -205,13 +235,21 @@ class write_sqd_sim_result_impl
         std::for_each(sim_result.charge_distributions.cbegin(), sim_result.charge_distributions.cend(),
```

Comment:
Unfortunately, there is no guarantee that the order will always be the same, since the exhaustive simulation starts with unordered maps, which are then converted to vectors. 
I just saw it because it still fails on my machine due to this problem. Thanks for your efforts!

Reply:
Ah, I see. The surfaces themselves are also unordered. Alright, we can fix that easily enough.